doc/classes/AnimatedSprite2D.xml

```diff
@@ -28,8 +28,8 @@
 		</method>
 	</methods>
 	<members>
-		<member name="animation" type="StringName" setter="set_animation" getter="get_animation" default="&amp;&quot;default&quot;">
-			The current animation from the [code]frames[/code] resource. If this value changes, the [code]frame[/code] counter is reset.
+		<member name="animation" type="StringName" setter="set_animation" getter="get_animation" default="@&quot;default&quot;">
```

Comment:
Won't doctool revert that change?

Reply:
Yes: https://github.com/godotengine/godot/runs/6413040535?check_suite_focus=true

Reply:
Fixed, it was a rebasing mistake.